flake.nix

```
@@ -65,7 +65,7 @@
       flake.outputsFromNickel = path: inputs: {
         systems ? flake-utils.lib.defaultSystems,
         lockFileContents ? {
-          nickel-nix = self.lib.x86_64-linux.lockFileContents;
+          nickel-nix = "${./lib}/nix.ncl";
```

```suggestion
          nickel-nix = "${./.}/lib/nix.ncl";
```

Otherwise `${./lib}` makes a separate copy of `lib` in the store that is less likely to be present - it will be copied only when this value is evaluated, not as part of flake dependency resolution.